.openpublishing.redirection.json

```diff
@@ -0,0 +1,14 @@
+{
+    "redirections": [
+      {
+        "source_path": "archive/graph/providers/InteractiveProviderBehavior.md",
+        "redirect_url": "/graph/providers/interactiveproviderbehavior",
+        "redirect_document_id": false
+      },
+      {
+        "source_path": "archive/graph/providers/MockProviderBehavior.md",
+        "redirect_url": "/graph/providers/mockproviderbehavior",
+        "redirect_document_id": false
+      }
+    ]
```

Comment:
We should redirect the MsalProvider too, eh?

Reply:
Oh, I did rename the MsalProvider, WindowsPRovider, and IProvider files. I'll add them to redirect as well

Reply:
Fixed!

Reply:
I found some details on "redirect_document_id" property:

> redirect_document_id indicates whether you would like to keep the document ID from the previous file. The default is false. Use true if you want to preserve the ms.documentid attribute value from the redirected article. If you preserve the document ID, data, such as page views and rankings, will be transferred to the target article. Do this if the redirect is primarily a rename, and not a pointer to different article that only covers some of the same content.

So based on this info, I also flipped all of the "return_document_id" values to `true` because they are caused by file renames, and not pointers to different articles.